packages/sdks-tests/src/specs/index.ts

```diff
@@ -126,6 +127,7 @@ export const PAGES: Record<string, Page> = {
   '/link-url': { content: linkUrl },
   '/symbol-with-input-binding': { content: symbolWithInputBinding },
   '/content-bindings': { content: contentBindings },
+  '/content-input-bindings': { content: contentInputBindings },
```

Comment:
for this to work with `gen1-react` you'll need to do

```suggestion
  '/content-input-bindings': { content: contentInputBindings, isGen1VisualEditingTest: true },
```

for more info see

https://github.com/BuilderIO/builder/blob/157e72f90522a53a41a26bed75a61ed05d2d6c3b/packages/sdks-tests/src/specs/index.ts#L103-L110